internal/daemon/controller/handlers/aliases/alias_service.go

```diff
@@ -49,6 +50,8 @@ var (
 	)
 )
 
+const aliasTypeTarget = "target"
```

Comment:
Should this also be used in createInRepo to switch on what type of alias to create and then call a create in repo method for that type? If not, I think we should add a comment to createInRepo to note that we'll need to update it when we add other alias types.